modules/cloud-run-v2/variables.tf

```diff
@@ -86,6 +92,10 @@ variable "eventarc_triggers" {
     service_account_create = optional(bool, false)
   })
   default = {}
+  validation {
+    condition     = var.eventarc_triggers.audit_log == null || (var.eventarc_triggers.audit_log != null && (var.eventarc_triggers.service_account_email != null || var.eventarc_triggers.service_account_create))
+    error_message = "When setting var.eventarc_triggers.audit_log provide either service_account_email or set service_account_create to true"
```

Comment:
I disabled EventArc end-to-end tests because you need to run apply, wait and then rerun apply (but before I got there, I got other error messages which remind me of this requirement). PubSub end-to-end tests work without specifying SA.